gravatar/src/main/java/com/gravatar/services/HttpException.kt

```diff
@@ -9,6 +9,7 @@ import retrofit2.Response
  * [message] The HTTP status message.
  */
 public class HttpException internal constructor(response: Response<*>) : RuntimeException() {
+    internal val rawErrorBody: String? = response.errorBody()?.string()
```

Comment:
You can only read the error body once so we need it here if we want to expose it and keep the `message` property.